### DISCUSSION TITLE
Let me make a helpful PSA for you all.

### DISCUSSION THREAD

**Discussion by ehunt34** [3kh0/ext-remover#262] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Don't ask questions here. Don't ask when this is going to be updated. Don't ask other ways to bypass.

Figure it out yourself.

Echo has no idea what they are doing. They are just waiting on discord for something to be found by people who are doing the work to find these exploits. Be one of those people.

You might be asking, "How do I do that?" or, "How do I find exploits?". That is actually very simple.

In Computer Science, programming is just transferring real life solutions to code. Knowing the language is just a part of it. Find out what you want to do. Then, break it into smaller steps. For example, logging into the playstore with a different account. You can break it down to finding out how to login, how to switch, then how to install blocked apps.

Obviously it is not as simple as that. You still need to find exploits to use. Solution: Research. A really good place to find them is bugs.chromium.org. Using that you can find things from graphical glitches to root escalation exploits. Neat stuff.

Don't ask here however. Echo and Yeet will make sure you don't get your freedom, and if they take this down that proves my point. Find it yourselves. Might make a career. Hell, Google will PAY you if you find something groundbreaking. Just don't ask here. You won't get answers.

**Comment by Fallenmoon8080** [comment]
This website: https://discord.gg/Bc6qmQ8g is THE place for unblockers and technical geeky stuff🤓

**Comment by 3kh0** [comment]
TL;DR, dont be dum

**Comment by ILikeBypassing** [comment]
"this is actually very simple"
oh ok ^


"obviously it is not as simple as that"
;-;

**Reply by ehunt34** [reply]
Simple in concept, not in execution.

**Reply by Aka-but-nice** [reply]
Man I speak english, but found one. Here's what you have to do. Make a bookmark with chrome://restart. Next, click on it and your Chromebook should restart, AS SOON AS IT COMES BACK UP CLICK THE CALENDER AT THE BOTTOM OF THE SCREEN and it will restart again then everything will be unblocked if you don't have a calendar at the bottom of the screen, go to chrome://flags and look up a calendar and enable anything that's there. It works for me if it doesn't work for you, I'm sorry but this works!
